contrib/kitti_poses_and_timestamps_to_trajectory.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from evo.core.trajectory import PoseTrajectory3D
+from evo.tools import file_interface
+import numpy as np
+
+DESC = "Combine KITTI poses and timestamps files to a TUM trajectory file"
+
+
+def kitti_poses_and_timestamps_to_trajectory(poses_file, timestamp_file):
+    pose_path = file_interface.read_kitti_poses_file(poses_file)
+    raw_timestamps_mat = file_interface.csv_read_matrix(timestamp_file)
+    error_msg = ("timestamp file must have same row with KITTI poses file")
+    if len(raw_timestamps_mat) > 0 and len(raw_timestamps_mat[0]) != 1 and len(timestamps) != pose_path.num_poses:
```

Comment:
There is no variable called `timestamps` and there should be an "or" instead of "and". Suggestion:
```suggestion
    if len(raw_timestamps_mat) > 0 and len(raw_timestamps_mat[0]) != 1 or len(raw_timestamps_mat) != pose_path.num_poses:
```

Reply:
After testing I add this, so I didn't find this error.